### ISSUE TITLE
Incorrect indentation of characters and dialogue when exporting PDFs where dialogue pieces are indented

### ISSUE THREAD

**Issue by rickschubert** [piersdeseilligny/betterfountain#75]
Given I open the Brick & Steel screenplay (in `src\test\scripts\brick_and_steel_crlf.fountain`)
When I export the screenplay to PDF using the command "Export screenplay PDF"
Then the indentation in the below scene is incorrect

```
INT. TRAILER HOME - DAY

This is the home of THE BOY BAND, AKA DAN and JACK.  They too are drinking beer, and counting the take from their last smash-and-grab.  Money, drugs, and ridiculous props are strewn about the table.

			JACK
		(in Vietnamese, subtitled)
	*Did you know Brick and Steel are retired?*

			DAN
	Then let's retire them.
	_Permanently_.

Jack begins to argue vociferously in Vietnamese (?), But mercifully we...
```

This is most likely happening due to the indentation of the dialogue and character markers. I assume that it _should_ be indented correctly though since "Brick & Steel" is one of the fountain example scripts and because other PDF converters handle it. Below is a screenshot comparing afterwriting.com (left) vs. betterfountain (right):

![image](https://user-images.githubusercontent.com/25436915/88326674-33344d00-cd1e-11ea-989a-6f9d71fb3a4c.png)

**Comment by piersdeseilligny** [comment]
So I'm currently on holiday until the 6th of August, but if you have time / want to submit a pr before then I'm assuming the bug can be fixed by slightly adjusting the regexes of `afterparser.ts`, which probably don't allow dialogue to start with whitespaces (and so it's just parsed as action). I assume they don't align correctly on the live preview either?
________________________________

**Comment by rickschubert** [comment]
Thanks for the explanation! I hadn't checked the live preview, and there it actually works. (That new dynamic switching feature is really amazing by the way, fantastic job on this one.) 

This issue doesn't distress me personally, I just wanted to raise it to contribute to the project. Feel free to work on it whenever you can or to close it if undesired. Thanks!

**Comment by piersdeseilligny** [comment]
Ah, good to know - there must be an issue with the pdf printer somewhere then. Thanks for the issue!

**Comment by piersdeseilligny** [comment]
So i'm not entirely sure why this wasn't happening in afterwriting but it was in betterfountain, seeing as the betterfountain parser is built on the afterwriting one. Regardless, it's been fixed in v1.6.6, which now trims spaces and tabs before all tokens except action ones (as it should do, according to the fountain.io spec)

![image](https://user-images.githubusercontent.com/42496508/90331351-e86fc300-dfb3-11ea-891e-95fb25f64a19.png)

**Comment by rickschubert** [comment]
Great, thanks!
